packages/frontity-org-theme/src/processors/borders.ts

```diff
@@ -0,0 +1,50 @@
+import { css } from "frontity";
+
+function createBordersProcessors(obj: {
+  [prop: string]: ReturnType<typeof css>;
+}) {
+  return Object.entries(obj).map(([className, style]) => ({
+    name: "borders",
+    test: element => {
+      return (
+        element.type === "element" &&
+        element.props.className?.split(/ +/).includes(className)
+      );
+    },
+
+    process: element => {
```

Comment:
Same here than `test`. Also, you have to change `process` by `processor`! (more API changes 🙈)  We decided to change that prop name because of a [weird error](https://github.com/frontity/frontity/pull/327#issuecomment-592599048) that appeared for the ES5 bundle.